exchanges/binance/ratelimit_test.go

```diff
@@ -0,0 +1,47 @@
+package binance
+
+import (
+	"testing"
+
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+)
+
+func TestRateLimit_Limit(t *testing.T) {
+	symbol := "BTC-USDT"
+
+	testTable := map[string]struct {
+		Expected request.EndpointLimit
+		Limit    request.EndpointLimit
+	}{
+		"All Orderbooks Ticker": {Expected: limitOrderbookTickerAll, Limit: bestPriceLimit("")},
```

Comment:
Just missing one for limitHistoricalTrades and then this LGTM

Reply:
Missing that one, as well as `limitPriceChangeAll` and `limitOrdersAll`, which don't have any helper function used with them. Went to add them, but skipped since the expected/actual would be hardcoded to the same value. I'll define a separate test to cover those 3 cases